README.rst

```diff
@@ -189,13 +192,11 @@ Spanner doesn't support it. For example:
 Schema migrations
 ~~~~~~~~~~~~~~~~~
 
-Spanner has some limitations on schema changes which you must respect:
+There are some limitations on schema changes to consider:
```

Comment:
👍

Reply:
Looks like this one has already been fixed.